src/test/java/teammates/test/cases/webapi/GetInstructorPrivilegeActionTest.java

```diff
@@ -20,16 +31,236 @@ protected String getRequestMethod() {
+
+        String[] sessionParams = {
+                Const.ParamsNames.COURSE_ID, instructor1ofCourse1.getCourseId(),
+                Const.ParamsNames.SECTION_NAME, student1InCourse1.getSection(),
+                Const.ParamsNames.FEEDBACK_SESSION_NAME, session1ofCourse1.getFeedbackSessionName(),
+        };
+
+        a = getAction(sessionParams);
+        response = (InstructorPrivilegeData) getJsonResult(a).getOutput();
+
+        assertTrue(response.isCanModifyCourse());
+        assertTrue(response.isCanModifyStudent());
+        assertFalse(response.isCanModifyInstructor());
+        assertFalse(response.isCanModifySession());
+
+        assertTrue(response.isCanViewStudentInSections());
+
+        assertFalse(response.isCanModifySessionCommentsInSections());
+        assertTrue(response.isCanViewSessionInSections());
+        assertTrue(response.isCanSubmitSessionInSections());
+
+        ______TS("Request with course id,"
+                + " section name and invalid session name, return section level privilege.");
+
+        String[] invalidSessionParams = {
+                Const.ParamsNames.COURSE_ID, instructor1ofCourse1.getCourseId(),
+                Const.ParamsNames.SECTION_NAME, student1InCourse1.getSection(),
+                Const.ParamsNames.FEEDBACK_SESSION_NAME, "invalid session",
+        };
+
+        a = getAction(invalidSessionParams);
+        response = (InstructorPrivilegeData) getJsonResult(a).getOutput();
+
+        assertTrue(response.isCanModifyCourse());
+        assertTrue(response.isCanModifyStudent());
+        assertFalse(response.isCanModifyInstructor());
+        assertFalse(response.isCanModifySession());
+
+        assertTrue(response.isCanViewStudentInSections());
+
+        assertFalse(response.isCanModifySessionCommentsInSections());
+        assertFalse(response.isCanViewSessionInSections());
+        assertFalse(response.isCanSubmitSessionInSections());
+
+        ______TS("Request with course id and invalid section name, return general privilege.");
+
+        String[] invalidSectionParams = {
+                Const.ParamsNames.COURSE_ID, instructor1ofCourse1.getCourseId(),
+                Const.ParamsNames.SECTION_NAME, "invalid section",
+        };
+
+        a = getAction(invalidSectionParams);
+        response = (InstructorPrivilegeData) getJsonResult(a).getOutput();
+
+        assertTrue(response.isCanModifyCourse());
+        assertTrue(response.isCanModifyStudent());
+        assertFalse(response.isCanModifyInstructor());
+        assertFalse(response.isCanModifySession());
+
+        assertFalse(response.isCanViewStudentInSections());
+
+        assertFalse(response.isCanModifySessionCommentsInSections());
+        assertFalse(response.isCanViewSessionInSections());
+        assertFalse(response.isCanSubmitSessionInSections());
+
+        ______TS("Request with course id and invalid session name, return general privilege.");
```

Comment:
The description is the same as L170?

Reply:
One is session and another is section. But the test is the same... , already fixed